BCStudentSoftwareDevTeam/celts
# Background check

This PR addresses issue #190. 

We implemented the logic that records if the student has passed the background check. We created a model calls backgroundCheck that has the user as the ForeignKey Field and a boolean field that indicates if that user passed the background check. 

We also have a simple select UI that is located on the User Profile page that indicates if the student has completed the background check. Selecting  Yes or No just records if the user has passed the background check. There is no further logic that relates to banning yet. 

![background check](https://user-images.githubusercontent.com/59736834/141821822-57a5a2ed-5b5e-435d-af91-b6f9cf056699.jpg)

Test: 
1. run tests. All tests in test_volunteer should pass
2. navigate to a student profile and the page that looks like above should show up. select Yes/No and check the backgroundcheck table to see if the passbackgroundCheck column corresponds to what you selected.
